lib/decompress/zstd_ddict.rs

```diff
@@ -241,6 +245,11 @@ pub unsafe extern "C" fn ZSTD_createDDict(
         ZSTD_customMem::default(),
     )
 }
+
+/// Create a digested dictionary, to start decompression without startup delay.
+///
+/// Dictionary content is simply referenced, it will be accessed during decompression.
+/// `dictBuffer` must outlive `DDict` (`DDict` must be freed before `dictBuffer`)
```

Comment:
```suggestion
/// `dictBuffer` must outlive [`DDict`] ([`DDict`] must be freed before `dictBuffer`)
```

This will create nice links in the docs. Also in general we'll eventually want the docs to look like https://docs.rs/libz-rs-sys/0.5.2/libz_rs_sys/fn.inflateInit2_.html.